docs/microservices.json

```diff
@@ -320,7 +393,7 @@
             "spec": null,
             "dependencies": [
                 {
-                    "id": "idam-web",
+                    "id": "authentication-web",
```

Comment:
probably idam-app as well... not sure though

Reply:
yup, I think you're right